spec/hackerone/client/activity_spec.rb

```diff
@@ -50,6 +50,37 @@
       expect(activity.bounty_amount).to eq 0
       expect(activity.bonus_amount).to eq 0
     end
+
+    it 'throws an error when bounty amount or bonus amount is malformed' do
+      example = {
+        'type' => 'activity-bounty-awarded',
+        'attributes' => {
+          'bounty_amount' => 'steve',
+          'bonus_amount' => 'harvey'
+        }
+      }.with_indifferent_access
+
+      activity = HackerOne::Client::Activities.build example
+
+      expect{ activity.bounty_amount }.to raise_error "Improperly formatted bounty amount"
+      expect{ activity.bonus_amount }.to raise_error "Improperly formatted bonus amount"
+    end
+
+    it 'returns 0 when bounty amount or bonus amount are malformed with lenient mode' do
+      ENV['HACKERONE_CLIENT_LENIENT_MODE'] = 'true'
```

Comment:
This setting will persist across tests and may result in seemingly random failures in the future. Let's add an `ENV.delete("HACKERONE_CLIENT_LENIENT_MODE")` to a [`before(:each)` block](https://relishapp.com/rspec/rspec-core/v/2-0/docs/hooks/before-and-after-hooks#define-before(:each)-block)